### ISSUE TITLE
Parsing CMYK percentages

### ISSUE THREAD

**Issue by kvz** [One-com/one-color#25]
Hi,

Imagemagick can output CMYK in the following format:

```bash
$ convert 9999716688CX.EPS -scale 1x1\! -format 'average_color::: %[pixel:u]' info:-
cmyk(1.95468%,3.82086%,5.06294%,11.3802%)
```

Passing that to onecolor `2.4.2`, gives:

```bash
      hex = onecolor("cmyk(1.95468%,3.82086%,5.06294%,11.3802%)").hex();
                                                                  ^
TypeError: Object false has no method 'hex'
```

Ideally, I could pass the `cmyk(1.95468%,3.82086%,5.06294%,11.3802%)` string straight into one-color without having to do any conversion on it myself first.

Any chance a future version of one-color might support that?

**Comment by Munter** [comment]
I don't have any objections to generally improving the parsing logic to cover more cases. This one should probably be late in the chain of syntaxes to check, just to keep up performance on the most common ones, as this seems to be an edge case

**Comment by Munter** [comment]
So somewhere around [here](https://github.com/One-com/one-color/blob/master/lib/color.js#L68) I would put in a new regex match that reuses the `[channelRegExp.source](https://github.com/One-com/one-color/blob/master/lib/color.js#L68)` that we also use to detect channels in the other color spaces.

**Comment by kvz** [comment]
Eager to move forward with my project, I found [this](https://github.com/substack/parse-color/blob/master/index.js#L14) which seems to already do what I need.

That said your project seems to be more feature-rich, so if something similar would be added, we'd probably switch in a heartbeat as I expect our requirements to go up.

**Comment by Munter** [comment]
Available in 2.5.0

**Comment by kvz** [comment]
:100: Thanks!
